CHANGES.md

```diff
@@ -139,7 +139,7 @@ Tactics
 - The syntax of the `autoapply` tactic was fixed to conform with preexisting
   documentation: it now takes a `with` clause instead of a `using` clause.
 
-
+- Modes are now taken into account by `typeclasses eauto` for local hypotheses.
```

Comment:
Which milestone did you intend this to be in? If it's really 8.11, then this location in changes is not consistent.

Reply:
In fact, I think we should target 8.10 as this is an important bugfix IMHO.